playbooks/vm_deploy.yaml

```diff
@@ -0,0 +1,40 @@
+# Copyright (C) 2023, RTE (http://www.rte-france.com)
+# SPDX-License-Identifier: Apache-2.0
+
+# Ansible playbook that create and start a VM.
+
+---
+
+# Fetch and create VM config file
+- hosts: hypervisors
+  name: Create and start guest{{ vm_id }}
+  vars:
+    - image_name: vm
+    - vm_id: 3
+    - cpu_set: 5
```

Comment:
I preferred using the cpu 6 for cpu_set. Because the 5 is not inside the RT slice in all our machines.